src/pmdas/bpf/modules/module_helpers.c

```diff
@@ -0,0 +1,25 @@
+#include "module_helpers.h"
```

Comment:
Since there's the only one small function here, we could simplify the build by making fill_instids_log2 a static inline in module.h and getting rid of module_helpers.[ch] entirely.

This would make things slightly easier for folks building modules.

Reply:
Sure. I've done this in a06b3a403.